opt_einsum/tests/test_paths.py

```diff
@@ -135,7 +132,6 @@ def test_explicit_path():
 
 
 def test_path_optimal():
-
```

Comment:
Do these new lines come from a black version bump?

Reply:
I believe so, noticed this in a few other libraries as well.